### PR TITLE
master-lps-99546

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-99546

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/PageEditor.macro [hunk 1]
```diff
@@ -311,6 +311,12 @@ definition {
 			value1 = "${commentReply}");
 	}
 
+	macro resolveComment {
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/PageEditor.macro]: This is ok for now. We may need to rename this or create a new macro when resolving comments when the "Show Resolved Comments" checkbox is enabled. We should assert in that case that the reply button is disabled.

image

**Comment by austinchiang** [comment]
Hey @liuxingye I removed the edit and reply to comment stuff because we covered those in other tests. The main thing we are testing here is the ability to resolve a comment so let's just focus on that. Please let me know if this is done in error. Thanks!

**Comment by austinchiang** [comment]
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/76886

**Pull request closed without merging**